changes/555.housekeeping

```diff
@@ -0,0 +1 @@
+Updated to mkdocs-ansible-collection 1.0.0
```

Comment:
Please change the filename to `555.dependencies`

Reply:
Done